dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
The swagger file from last sprint wasn't stored anywhere. Wasn't sure if we wanted to hold on to it or not or if we just were going to update the README